app/src/main/java/com/android/sample/ui/theme/Color.kt

```diff
@@ -16,3 +16,11 @@ val Grey = Color(0xFFD9D9D9)
 val Blue = Color(0xFFABC2FD)
 val DarkBlue = Color(0xFF525A91)
 val DarkBlue2 = Color(0xB5525A91)
+val PastelBlue = Color(0xFFFFCDD2)
+val PastelRed = Color(0xFFE3F2FD)
+val Yellow = Color(0xFFFFFF00)
+val Black = Color(0xFF000000)
+val Red = Color(0xFFFF0000)
+val DarkGray = Color(0xFF444444)
+val Gray = Color(0xFF888888)
+val LightGray = Color(0xFFCCCCCC)
```

Comment:
There's Gray and Grey, another name would've been better

Reply:
Fixed.